src/main/resources/maven_templates/maven-failsafe-plugin.xml

```diff
@@ -0,0 +1,21 @@
+<plugin>
+    <groupId>org.apache.maven.plugins</groupId>
+    <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
And the same for this plugin. The different project might not fit into this template